main.go

```diff
@@ -157,49 +159,49 @@ func getVars(config []byte) ([]string, error) {
 
 // updateConfig looks in the config file for variables that should be updated from the local environment, swaps out
 // the value, and sends the new config file to AWS AppConfig
-func updateConfig(params AppConfigParams, cfgBytes []byte) error {
+func updateConfig(params AppConfigParams, cfgBytes []byte) ([]byte, error) {
```

Comment:
Do you want to rename cfgBytes here as well?

Reply:
Yes, thanks